src/antsibull_build/build_changelog.py

```diff
@@ -492,6 +492,12 @@ def append_removed_collections(
                 f"(previously included version: {collection_version})",
                 text_format=TextFormat.RESTRUCTURED_TEXT,
             )
+        section.ensure_paragraph_break()
+        section.add_text(
+            "You can still install a removed collection manually with"
+            " ``ansible-galaxy collection install <name-of-collection>``.",
```

Comment:
Is it worth adding a reference in the docs here: https://docs.ansible.com/ansible/latest/collections_guide/collections_installing.html

Maybe we could then add a reference to the docs to this section too? Or maybe that's overkill...

Reply:
This is probably not really worth mentioning in that doc or linking from here. It was just a thought.